src/main/scala/io/iohk/praos/crypto/package.scala

```diff
@@ -59,5 +59,44 @@ package object crypto {
   /**
     * The seed for a PRNG.
     */
-  type Seed = Int
+  type Seed = RandomValue
+
+  /**
+    * Operation used to "combine" seeds. According to the Praos Formalization, it should be associative.
+    * @note Currently implemented as the number obtained by concatenating the binary representation of both arguments.
+    */
+  def combineSeeds(x: Seed, y: Seed): Seed = {
+    Integer.parseInt(Integer.toBinaryString(x) ++ Integer.toBinaryString(y), 2)
```

Comment:
Maybe this function should always return a positive number? We discussed with Mirko that it might be useful to do so as the random number AFAIK should always be positive (as in [here](https://github.com/atixlabs/praos_test/pull/6/files#diff-3f124fa27be1fc3a524517d6ca434401R15)). Do you know if maybe there is any place were that is not the case?

Reply:
How could the concatenation of the binary representation of two positive numbers yield a negative number (apart from `Integer` overflow)? Please keep in mind that a `Seed` is a type alias for `RandomNumber` (which is assumed to be a positive number). Am I missing something?

Reply:
You are right, I thought `Integer.parseInt` worked differently. The case that I was thinking that could bring issues is for example `combineSeeds(Int.MaxValue, 1)` which seems to cause an exception, should we maybe address that case here?

Reply:
As far as I understand, Mirko has already changed the implementation to `abs(x+y)`, correct?